internal/io/file/constant.go

```diff
@@ -22,6 +22,13 @@ const (
 	LINES_TYPE FileType = "lines"
 )
 
+const (
+	ZLIB  = "zlib"
```

Comment:
I am not sure if zlib/flate is needed. I think gzip use those compression algorithm and add some headers to the file?